src/components/common/ItemValues/GraphItemValue.tsx

```diff
@@ -15,22 +15,32 @@ export const GraphItemValue: React.FC<IItemValue> = ({ empty, onSelect, selected
     return null;
   }
 
+  const [isHovered, setIsHovered] = useState(false);
   const cardClass = !!selected?.id && value?.id === selected.id ? css?.selected : null;
 
   const onClick = (e: any) => {
     if (!!onSelect) onSelect(e, value);
+
   };
 
-  return <Card
+  return (
+    <Card
+    onMouseEnter={() => setIsHovered(true)}
+    onMouseLeave={() => setIsHovered(false)}
```

Comment:
I don't think there should be any reason to use hover state this way.
The CSS can detect on hover with a mouse.

see:
- https://www.w3schools.com/cssref/sel_hover.php

Mouse Enter / Mouse Leave should not be necessary.
I think the entire State logic added here should be removed.